lms/djangoapps/support/views/contact_us.py

```diff
@@ -18,8 +18,7 @@ class ContactUsView(View):
     def get(self, request):
         context = {
             'platform_name': configuration_helpers.get_value('platform_name', settings.PLATFORM_NAME),
-            'zendesk_api_host': settings.ZENDESK_URL,
-            'access_token': 'DUMMY_ACCESS_TOKEN',  # LEARNER-3450
+            'zendesk_proxy_url': settings.CMS_BASE,
```

Comment:
Why is this the CMS? Should there be an LMS endpoint?

Reply:
It'd be easy enough to add one - simply add a line like https://github.com/edx/edx-platform/pull/16769/files#diff-41bc677b7aca064fe1d4a263faad5b28R156 to `lms/urls.py`.

We previously only made a CMS endpoint because the initial usage was from CMS, but that doesn't mean every caller needs to hit that exact url.

Reply:
Also, since this variable is named `zendesk_proxy_url` (as opposed to `zendesk_proxy_host`), I think it should include the full URL like this:

`"https://{}/zendesk_proxy/v0".format(settings.LMS_BASE)`

Then the js code can just use the value directly without needing to construct the final URL on the fly.

Reply:
Sorry it was my bad. I believe after changing this, I will update this.

Reply:
@efischer19 thanks for clarifying this, I will add similar in LMS and use that instead of this.